doc/source/reference/introduction.rst

```diff
@@ -224,7 +224,7 @@ many that we have not developed yet too.  If you implement a
 graph algorithm that might be useful for others please let 
 us know through the 
 `NetworkX Google group <http://groups.google.com/group/networkx-discuss>`_
-or the Github `Developer Zone <http://networkx.github.io/networkx/networkx>`_.
+or the Github `Developer Zone <https://www.github.com/networkx/networkx>`_.
```

Comment:
I'm not sure it matters since GitHub manages it, but the `www` gets stripped when the address is resolved...so it's not technically necessary to include it in the link.